### ISSUE TITLE
Undefined variables in Task.py

### ISSUE THREAD

**Issue by pllim** [ejeschke/ginga#542]
In `WorkerThread` class in `Task.py`:
```python
    def stop(self):
        # Put termination sentinal on queue
        self.queue.put((priority, task))  # <-- priority and task are undefined!
        self.ev_quit.set()
```

**Comment by ejeschke** [comment]
Fixed in aef1e3817efaec00b52733b713cffc2862111a6e.

I think this error probably never showed up because the sentinal `ev_quit` is used to stop the thread pool and this is noticed after the queue wait timeout.

**Comment by pllim** [comment]
Thanks for the quick fix!
